test/integration/framework/master_utils.go

```diff
@@ -190,6 +190,7 @@ func startMasterOrDie(masterConfig *master.Config, incomingServer *httptest.Serv
 	masterConfig.ExtraConfig.VersionedInformers = informers.NewSharedInformerFactory(clientset, masterConfig.GenericConfig.LoopbackClientConfig.Timeout)
 	m, err = masterConfig.Complete().New(genericapiserver.NewEmptyDelegate())
 	if err != nil {
+		klog.Errorf("error in bringing up the master: %v", err)
```

Comment:
One last nit: I'd put a comment here explaining why we double-log. Otherwise, somebody helpful is going to come along and de-duplicate it. :smile: